src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -889,9 +889,10 @@ bool ViewProviderSketch::mouseButtonPressed(int Button, bool pressed, const SbVe
                     }
                     Mode = STATUS_NONE;
                     return true;
-                case STATUS_SKETCH_StartRubberBand: // a single click happened, so clear selection
+                case STATUS_SKETCH_StartRubberBand: // a single click happened, so clear selection unless user hold control.
+                    if (!(QApplication::keyboardModifiers() == Qt::ControlModifier))

```

Comment:
@PaddleStroke This suffers from the same problem but the Inventor API can't be used here. So, the check should be: `if (!(QApplication::keyboardModifiers() & Qt::ControlModifier))`